# /app/onyx/connectors/zendesk/connector.py v0.19.0
diff --git a/app/onyx/connectors/zendesk/connector.py b/app/onyx/connectors/zendesk/connector.py
index 17df88e..a1af4f4 100644
--- a/app/onyx/connectors/zendesk/connector.py
+++ b/app/onyx/connectors/zendesk/connector.py
@@ -96,6 +96,7 @@ def _get_tickets(
     client: ZendeskClient, start_time: int | None = None
 ) -> Iterator[dict[str, Any]]:
     params = {"start_time": start_time} if start_time else {"start_time": 0}
+    params["exclude_deleted"] = "true"
 
     while True:
         data = client.make_request("incremental/tickets.json", params)
@@ -368,8 +369,10 @@ class ZendeskConnector(LoadConnector, PollConnector):
                 try:
                     ticket = next(ticket_generator)
 
-                    # Check if the ticket status is deleted and skip it if so
-                    if ticket.get("status") == "deleted":
+                    if any(
+                        tag in ZENDESK_CONNECTOR_SKIP_TICKET_TAGS
+                        for tag in ticket.get("tags", [])
+                    ):
                         continue
 
                     new_author_map, documents = _ticket_to_document(
